### DISCUSSION TITLE
请问参考文献部分为什么报错BibTeX: Cannot find 'thesis.bib'!？

### DISCUSSION THREAD

**Discussion by jpDeng111** [sjtug/SJTUThesis#644] [提问] [answered]
卡在这好几个小时了，
代码\printbibliography[heading=bibintoc]左边出现了warning符号：empty bibliography on input  line 68（这串代码在第68行）
但是原来的模板中没有这个警告
请问是出了什么问题呢？

**Comment by jpDeng111** [comment]
我中间把thesis.bib移到了别的文件夹之下，但是后来又移动回bibdata这个文件夹下了，用overleaf写的论文

**Comment by jpDeng111** [comment]
然后sjtusetup中的这两行是这样的（这部分我之前也没动过）：
% 参考文献支持宏包
\usepackage[backend=biber,style=gb7714-2015,gbnamefmt=lowercase,gbpub=false,gbpunctin=false]{biblatex}
% 导入参考文献数据库
\addbibresource{bibdata/thesis.bib}

**Comment by jpDeng111** [comment]
池先生（如果是池的话），谢谢你的回复！我确认了一下，确实是XeTex，并且我又试了一下，发现把
\addbibresource{bibdata/thesis.bib}
这里面的文件夹和文件重新创建并且改了名（bibdata1和thesis2.bib）之后，发现编译出来还是BibTeX: Cannot find 'thesis.bib，说明\addbibresource{}没起作用




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Alex ***@***.***&gt;; 
发送时间: 2021年5月25日(星期二) 下午4:43
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [sjtug/SJTUThesis] 请问参考文献部分为什么报错BibTeX: Cannot find &#39;thesis.bib&#39;!？ (#644)





 
请检查 overleaf 的编译器是否是 XeTeX。

**Comment by skyzh** [comment, marked as answer]
感觉上是 Overleaf 的问题。Overleaf 有类似于清理缓存的功能吗？清理之后应该就可以用了。

**Reply by skyzh** [reply]
可以参考 https://www.overleaf.com/learn/how-to/Clearing_the_cache

**Reply by jpDeng111** [reply]
哦谢谢！我试试看

**Reply by jpDeng111** [reply]
解决了！重新复制粘贴导入模板就好了

**Reply by jpDeng111** [reply]
非常谢谢！
